test/WebJobs.Script.Tests/Middleware/HostWarmupMiddlewareTests.cs

```diff
@@ -81,11 +156,55 @@ public void ReadRuntimeAssemblyFiles_VerifyLogs()
             TestLoggerProvider testLoggerProvider = new TestLoggerProvider();
             testLoggerFactory.AddProvider(testLoggerProvider);
             ILogger<HostWarmupMiddleware> testLogger = testLoggerFactory.CreateLogger<HostWarmupMiddleware>();
-            HostWarmupMiddleware hostWarmupMiddleware = new HostWarmupMiddleware(null, new Mock<IScriptWebHostEnvironment>().Object, environment, new Mock<IScriptHostManager>().Object, testLogger);
+
+            HostWarmupMiddleware hostWarmupMiddleware = new HostWarmupMiddleware(null, new Mock<IScriptWebHostEnvironment>().Object, environment, new Mock<IScriptHostManager>().Object, testLogger, _rpcWorkerChannelManager, _functionsHostingConfigOptions);
+
             hostWarmupMiddleware.ReadRuntimeAssemblyFiles();
             // Assert
             var traces = testLoggerProvider.GetAllLogMessages();
             Assert.True(traces.Any(m => m.FormattedMessage.Contains("Number of files read:")));
         }
+
+        //[Fact]
```

Comment:
Did we mean to disable this test?

Reply:
Yes, it was work in progress. Added the test